spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe ApplicationHelper, type: :helper do
+  context "when checking status" do
+    def check_api_football_status
+      api_key = Rails.application.credentials.api_sports.football[:key]
+      api_football_status_url = "https://v3.football.api-sports.io/status"
+
+      ApplicationHelper::APIFootball.get(api_football_status_url, headers: { "x-apisports-key" => api_key })
```

Comment:
we should probably use a different approach here
1. use VCR cassettes to record the API response so we don't need to make external calls every time we run the specs, it is expensive to do so
2. We should mock the API response and stub the call to the API

we can have a call to work on this if you want